playbooks/roles/configure-backups/defaults/main.yml

```diff
@@ -0,0 +1,90 @@
+---
+
+backup_enabled: no             # Enable the role
+backup_remove: no               # Set yes for uninstall the role from target system
+backup_cron: yes                # Setup cron tasks for backup
+
+backup_user: root               # Run backups as user
```

Comment:
you can use any user for backup
f.e.:
```
backup_profiles:
  - name: mysql
    schedule: 0 3 * * *
    source: "mysql://"
    #target: "s3://s3-{{ region }}.amazonaws.com/{{ aws_backup_mysql_bucket }}/"
    target: "s3://s3.amazonaws.com/{{ aws_backup_mysql_bucket }}/"
    user: XXXXXX
```
....
owner: "{{ item.user|default(backup_user) }}"